frontend/src/pages/DriverInfo/DriverInfo.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Form, Icon, Button } from 'semantic-ui-react';
+import { Link } from "react-router-dom";
+
+import './DriverInfo.css';
+
+class DriverInfo extends React.Component {
+    constructor() {
+        super();
+
+        this.state = {
+            plate: '',
+            make: '',
+            model: '',
+            color: '',
+            capacity: 4,
+        };
+    }
+
+    postFetch = () => {
+        const userId = '5f2f0fdb12db250479914d5b';  //TODO: get currentUserID, implement onAuthStateChange
+        fetch(`http://localhost:3000/api/vehicles/${userId}`, {
```

Comment:
Remember to remove localhost urls for deployment

Reply:
got it, lemme make that change